impeller/renderer/prefix_sum_test.comp

```diff
@@ -31,7 +31,10 @@ void main() {
     value = input_data.data[ident];
   }
 
-  memory[ident] = value;
+  if (ident < BLOCK_SIZE) {
```

Comment:
Technically we need to use a function constant sizing of the memory size so that it exactly corresponds to the threadgroup size.